lib/routes/reuters/utilsUS.js

```diff
@@ -0,0 +1,71 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+const ProcessFeedUS = async (link) => {
```

Comment:
There is a function to get full text. Why isn't there full text in rss

Reply:
I did not intend to modify get-full-text function.  Somehow I left it as it is before my PR.  I did not contribute nor make change to it.

Actually I dont know how to get full text yet.

Reply:
> I did not intend to modify get-full-text function.  Somehow I left it as it is before my PR.  I did not contribute nor make change to it.
> 
> Actually I dont know how to get full text yet. 

But the output should be full text. Please fix

Reply:
Thanks for your guidance. The patches fit my needs quite well at the moment. The get-full-text will be scheduled to uncertain delay by my part.

Reply:
@LyleLee How about I fix the full text fetching and file a PR to your repo or this repo. Your attribution to each commits will be preserved

Reply:
That will be nice.